### PR TITLE
vmui: timezone select

### PR DESCRIPTION
**Pull request by Loori-R**
Provide feature Timezone select.

- You can select the timezone from "Settings".
- The browser provides the list of supported values (see [Intl.supportedValuesOf('timeZone')](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/supportedValuesOf))
- Selected value stores in `localStorage`
- #3075

### CODE REVIEW

File: app/vmui/packages/vmui/src/components/Chart/LineChart/LineChart.tsx [hunk 1]
```diff
@@ -163,6 +170,7 @@ const LineChart: FC<LineChartProps> = ({
 
   const options: uPlotOptions = {
     ...defaultOptions,
+    tzDate: ts => uPlot.tzDate(new Date(ts * 1e3), timezone),
```
Comment (dmitryk-dk) [inline app/vmui/packages/vmui/src/components/Chart/LineChart/LineChart.tsx:173]: Let's use dayjs everywhere

File: app/vmui/packages/vmui/src/utils/time.ts [hunk 2]
```diff
@@ -151,3 +146,31 @@ export const getRelativeTime = ({ relativeTimeId, defaultDuration, defaultEndInp
...
+    const zone = (region.match(/^(.*?)\//) || [])[1] || "unknown";
+    const utc = getUTCByTimezone(region);
+    const item = {
+      region,
+      utc,
+      search: `${region} ${utc} ${region.replace(/[/_]/gmi, " ")}`
+    };
+
+    if (!search || (search && regexp.test(item.search))) {
+      acc[zone] ? acc[zone].push(item) : acc[zone] = [item];
```
Comment (dmitryk-dk) [inline app/vmui/packages/vmui/src/utils/time.ts:171]: it is better to use if else combination instead of the ternary operator with assignment

**Review by dmitryk-dk — APPROVED** [review]
LGTM!
nit: some simple comments

**Review by valyala — APPROVED** [review]

File: app/vmui/packages/vmui/src/components/Chart/LineChart/LineChart.tsx (hunk 1 above)
Comment (valyala) [reply app/vmui/packages/vmui/src/components/Chart/LineChart/LineChart.tsx]: Why do you think dayjs is better to use in this particular place?

File: app/vmui/packages/vmui/src/utils/time.ts (hunk 2 above)
Comment (valyala) [reply app/vmui/packages/vmui/src/utils/time.ts]: Agreed - the `if ... else ...` code is easier to read and maintain than the trenary operator :)

**Comment by valyala** [comment]
@Loori-R , thanks for the pull request!

File: app/vmui/packages/vmui/src/components/Chart/LineChart/LineChart.tsx (hunk 1 above)
Comment (Loori-R) [reply app/vmui/packages/vmui/src/components/Chart/LineChart/LineChart.tsx]: I think it's better to have one approach to converting timezones everywhere. In our case, this is dayjs

**Pull request merged**